src/login_helper.js

```diff
@@ -61,6 +66,347 @@ function filterPastAndUpcomingTrips(travelerTrips) {
+  let startDateNode = document.createTextNode(
+    `Trip Start Date: ${formattedStartDate.toDateString()}`
+  );
+  let endDateNode = document.createTextNode(
+    `Trip End Date: ${formattedEndDate.toDateString()}`
+  );
+  startDateParagraph.appendChild(startDateNode);
+  endDateParagraph.appendChild(endDateNode);
+  document.querySelectorAll("section")[3].appendChild(startDateParagraph);
+  document.querySelectorAll("section")[3].appendChild(endDateParagraph);
+}
+
+function renderTripCost(destination, tripRequestData, allTrips) {
+  let costs = calculateTripRequestCosts(destination, tripRequestData, allTrips);
+  let totalCostNode = document.createTextNode(
+    `Total Estimated Trip Cost: $${costs[5].toFixed(2)}`
+  );
+  document.querySelectorAll("section")[3].appendChild(totalCostNode);
+}
+
+function confirmAndPostTrip(destination, tripRequestData, allTrips) {
+  getSingleTraveler(destination, tripRequestData, allTrips).then(function (
+    result
+  ) {
+    let values = getValuesFromInputs();
+    let tripRequestStartDate = values[1];
+    let tripRequestEndDate = values[2];
+    let traveler = result;
+    let userId = traveler.traveler.id;
+    let tripRequestId = allTrips.length + 1;
+    const data = JSON.stringify({
+      id: parseFloat(tripRequestId),
+      userID: userId,
+      destinationID: tripRequestData[0],
+      travelers: parseFloat(tripRequestData[2].substring(21)),
+      date: tripRequestStartDate.replace(/-/g, "/"),
+      duration: daysBetween(tripRequestStartDate, tripRequestEndDate),
+      status: "pending",
+      suggestedActivities: [],
+    });
+    fetch(
+      "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips",
+      {
+        method: "POST", // or 'PUT'
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: data,
+      }
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("Success:", data);
+      })
+      .catch((error) => {
+        console.error("Error:", error);
+      });
+  });
+}
+
+function renderconfirmTripButton() {
+  let confirmTripRequestButton = document.createElement("button");
+  confirmTripRequestButton.setAttribute("id", "confirm-trip-request-button");
+  confirmTripRequestButton.textContent = "CONFIRM TRIP";
+  document.querySelectorAll("section")[3].appendChild(confirmTripRequestButton);
+}
+
+function renderTripRequest(tripRequestData, destinationsArr, allTrips) {
+  let values = getValuesFromInputs();
+  let tripRequestDestination = values[0];
+  let tripRequestStartDate = values[1];
+  let tripRequestEndDate = values[2];
+  let i;
+  for (i = 0; i < destinationsArr.length; i++) {
+    if (tripRequestDestination === destinationsArr[i].destination) {
+      document.getElementById("request-form").style.display = "none";
+      renderTripsHeader("Your Requested");
+      document
+        .querySelectorAll("section")[3]
```

Comment:
again, all these hardcoded array instances would be more appropriate working with object attributes

the outputs of the getValuesFromInputs();